stdlib/bytesLabels.mli

```diff
@@ -281,7 +329,137 @@ val compare: t -> t -> int
 
 val equal: t -> t -> bool
 (** The equality function for byte sequences.
-    @since 4.05.0 *)
+    @since 4.03.0 (4.05.0 in BytesLabels) *)
+
+(** {1:unsafe Unsafe conversions (for advanced users)}
```

Comment:
There is a small issue here: ocamldoc reference space is flat. Thus the label conflicts with the one in `Bytes`.

Reply:
Are you sure of that ? As far as I can remember you always had to refer to sections from outside using `{!M.ref}`.